sos/report/plugins/openstack_instack.py

```diff
@@ -61,19 +61,7 @@ def setup(self):
         vars_any = [p in os.environ for p in [
                     'OS_TENANT_NAME', 'OS_PROJECT_NAME']]
 
-        # capture yaml files to define overrides
-        uc_config = configparser.ConfigParser()
-        try:
-            uc_config.read(UNDERCLOUD_CONF_PATH)
-            override_opts = ['hieradata_override', 'net_config_override']
-            for opt in override_opts:
-                p = uc_config.get(opt)
```

Comment:
I believe this part never worked.

`configparser.get` requires 2 arguments: section and option - here it passes just the option.